package.json

```diff
@@ -102,6 +102,7 @@
     "hat": "0.0.3",
     "helmet": "^3.20.0",
     "inxt-service-mailer": "internxt/mailer",
+    "@sendgrid/mail": "^7.2.1",
```

Comment:
I think this change should come with a change in yarn.lock?

try adding it with:

node 14  and `yarn install --ignore-engines` if you have any issue let me know